dpytools/config/README.md

```diff
@@ -0,0 +1,159 @@
+# dpytools: Config
+
+Configuration class to enforce and police python application settings.
+
+## Usage
+
+The from_env() method is used to define a config object and populate it with 
+a dictionary as input. The attributes of the created Config object are generated 
+from the contents of the dictionary. See below example.
+
+## Basic Example
+
+Say we want to instantiate a config object, setting our configuration with an environment variable.
+The environment variable will be called `MY_SERVER_ENDPOINT` and we want it to be a URL.
+
+```python
+# The environment variable we want to use for our config. The value is a string of a URL.
+MY_SERVER_ENDPOINT = "http://my-url.com"
```

Comment:
Might be better to do `os.environ["MY_SERVER_ENDPOINT"] = "http://my-url.com"` (same for the other examples) so you're actually setting an env var.